### PR TITLE
Update err message for signal name

### PR DESCRIPTION
**Pull request by alexseedkou**
## What changed?
Update the error message for signalName when it is empty. 
For more details please refer to the request: https://github.com/temporalio/temporal/issues/5544

## Why?

Update the error message to resolve the confusion when signalName is empty but not too long.

### CODE REVIEW

File: service/frontend/errors.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ var (
 	errWorkflowTypeTooLong                                = serviceerror.NewInvalidArgument("WorkflowType length exceeds limit.")
 	errWorkflowIDTooLong                                  = serviceerror.NewInvalidArgument("WorkflowId length exceeds limit.")
 	errSignalNameTooLong                                  = serviceerror.NewInvalidArgument("SignalName length exceeds limit.")
+	errSignalNameIsEmpty                                  = serviceerror.NewInvalidArgument("SignalName is empty.")
```
Comment (yycptt) [inline service/frontend/errors.go:56]: The error is already define on L35. `errSignalNameNotSet`

**Comment by yycptt** [comment]
Thanks for fixing the contribution!

**Comment by alexseedkou** [comment]
> Thanks for the contribution!



> service/frontend/errors.go

Sure, I will make the change and update the PR soon. Thank you!

File: service/frontend/errors.go (hunk 1 above)
Comment (alexseedkou) [reply service/frontend/errors.go:56]: Thanks for pointing that out. I will make a change and update it soon.

**Review by yycptt — APPROVED** [review]

**Pull request merged**